orttraining/orttraining/core/graph/gradient_schema_defs.cc

```diff
@@ -1046,6 +1046,12 @@ Example 4:
              "the case during training.",
              "T1",
              OpSchema::Optional)
+      .Input(3, "training_mode",
```

Comment:
The TrainableDropout op doesn't have a training_mode input. When building the TrainableDropoutGrad, this training_mode will be thus be left empty. 
In the kernel, an empty "training_mode" implies inference mode. Thus, we have introduced a bug.

Reply:
For TrainableDropoutGrad, I think it's better to leave the schema as it was. 
Can we use the template<bool is_trainable>trick to make the kernel work for both DropoutGrad and TrainableDropoutGrad.